index.html

```diff
@@ -1,232 +1,326 @@
+          you see and have a project you need coded, don't hestiate to contact
+          me.
+        </p>
+        <h3 class="letsconnect">LET'S CONNECT</h3>
+
+        <ul class="socialmedia-list">
+          <li class="socials">
+            <img src="images/Vector.png" alt="vector1" />
+          </li>
+          <li class="socials">
+            <img src="images/Linkedinicon.png" alt="vectorlink" />
+          </li>
+
+          <li class="socials"><img src="images/med.png" alt="med" /></li>
+          <li class="socials"><img src="images/cat.png" alt="cat" /></li>
+          <li class="socials"><img src="images/angelist.png" alt="ange" /></li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="container">
+      <section class="works-temp1 works1">
+        <img src="images/SnapshootPortfolio(1).png" class="pic-temp1" alt="" />
+        <div class="works-info1">
+          <h1 class="tonic">Tonic</h1>
+          <ul class="work-info">
+            <li class="canopy">CANOPY</li>
+            <li class="back-end-dev">
+              <img src="images/Counter(1).png" alt="" />
+              <span>Back End Dev</span>
+            </li>
+            <li class="back-end-dev">
+              <img src="images/Counter(1).png" alt="" />
+              <span>2015</span>
+            </li>
+          </ul>
+          <p class="proj-info">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="prog-lang">
+            <li class="lang-styles">HTML</li>
+            <li class="lang-styles">Css</li>
+            <li class="lang-styles">JavaScript</li>
+          </ul>
+          <button type="button" class="btn">See Projects</button>
+        </div>
+      </section>
+
+      <section class="works-temp1 works2">
+        <img
+          src="images/SnapshootPortfolio(2).png"
+          class="pic-temp1"
+          alt="temp1"
+        />
+        <div class="works-info2">
+          <h1 class="tonic">
+            Multi-Post <br />
+            Stories
+          </h1>
+          <ul class="work-info">
+            <li class="canopy">CANOPY</li>
+            <li class="back-end-dev">
+              <img src="images/Counter(1).png" alt="" />
+              <span>Back End Dev</span>
+            </li>
+            <li class="back-end-dev">
+              <img src="images/Counter(1).png" alt="" />
+              <span>2015</span>
+            </li>
+          </ul>
+          <p class="proj-info">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="prog-lang">
+            <li class="lang-styles">HTML</li>
+            <li class="lang-styles">Css</li>
+            <li class="lang-styles">JavaScript</li>
+          </ul>
+          <button type="button" class="btn">See Projects</button>
```

Comment:
- [ ] Everything is good here but I think just you need to add some margin/padding here to match the Figma design as much as possible 👌🏻👌🏻✨✨💯💯

- your design
![image](https://user-images.githubusercontent.com/58553711/179330194-5f45adf2-dbb5-4404-81eb-f96bf269967d.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/179330345-b445f6f2-5d30-4fc9-8553-63ba5a247026.png)